### PR TITLE
adding battery undock level to infremen

### PR DESCRIPTION
**Pull request by Jailander**

### CODE REVIEW

File: info_terminal/infremen/src/infremen.cpp [hunk 1]
```diff
@@ -97,9 +98,10 @@ uint32_t getMidnightTime(uint32_t givenTime)
 //parameter reconfiguration
 void reconfigureCallback(infremen::infremenConfig &config, uint32_t level) 
 {
-	ROS_INFO("Reconfigure Request: %lf %d %d %d %d", config.explorationRatio, config.minimalBatteryLevel, config.interactionTimeout, config.maxTaskNumber, config.taskDuration);
+    ROS_INFO("Reconfigure Request: %lf %d %d %d %d %d", config.explorationRatio, config.minimalBatteryLevel, config.batteryUndockLevel, config.interactionTimeout, config.maxTaskNumber, config.taskDuration);
```
Comment (marc-hanheide) [inline info_terminal/infremen/src/infremen.cpp]: Can I ask you to make sure you use the same indentation as in the rest of the file as a courtesy to the original author?

Comment (Jailander) [inline info_terminal/infremen/src/infremen.cpp]: It was qtcreator but I'll fix that as a courtesy to @gestom

**Comment by marc-hanheide** [comment]
cheers! Is this already compiled on Henry?

**Comment by Jailander** [comment]
yes, only needs to restart infoterminal

**Pull request merged**